cmd/routedns/main.go

```diff
@@ -716,3 +724,13 @@ func newIPBlocklistDB(l list, locationDB string, rules []string) (rdns.IPBlockli
 		return nil, fmt.Errorf("unsupported format '%s'", l.Format)
 	}
 }
+
+func printVersion() {
+	var versionString interface{} = rdns.BuildVersion
+	var buildNumber interface{} = rdns.BuildNumber
+	var buildTime interface{} = rdns.BuildTime
```

Comment:
Any reason to declare it as an `interface{}` here? Wouldn't it be possible to just do
```golang
  fmt.Println("Build:",  rdns.BuildVersion)
  ...
```

Reply:
We can change that